packages/postcss-reduce-initial/src/index.js

```diff
@@ -20,6 +20,9 @@ export default plugin('postcss-reduce-initial', () => {
 
     css.walkDecls((decl) => {
       const lowerCasedProp = decl.prop.toLowerCase();
+      if (resultOpts.ignore && !!~resultOpts.ignore.indexOf(lowerCasedProp)) {
```

Comment:
Let's put `writing-mode` in default config because it is broken in the most popular browser - chrome

Reply:
yeah, but what if they want to transform that ?

Reply:
Ref: https://bugs.chromium.org/p/chromium/issues/detail?id=507397

I think we should always ignore this now, ideally we should use `browserlist` for this, but most of chromium based browsers are affected, we can't handle all of them using `browserlist` :disappointed: